audio/wav/decode.go

```diff
@@ -215,16 +215,14 @@ chunks:
 			}
 			channelCount := int(buf[2]) | int(buf[3])<<8
 			switch channelCount {
-			case 1:
-				mono = true
-			case 2:
-				mono = false
+			case 1, 2:
+				mono = channelCount == 1
```

Comment:
IMO this new code is less clear and less explicit than before.

Reply:
Your codebase. Your call. :)